thirdparty/build-thirdparty.sh

```diff
@@ -566,6 +566,20 @@ build_flatbuffers() {
   cp libflatbuffers.a $TP_LIB_DIR/libflatbuffers.a
 }
 
+build_brotli() {
+    check_if_source_exist $BROTLI_SOURCE
+    cd $TP_SOURCE_DIR/$BROTLI_SOURCE
+    mkdir -p $BUILD_DIR
+    cd $BUILD_DIR
+    ${CMAKE_CMD} .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$TP_INSTALL_DIR
+    ${BUILD_SYSTEM} -j$PARALLEL
+    ${BUILD_SYSTEM} install
+
+    mv -f $TP_INSTALL_DIR/lib/libbrotlienc-static.a $TP_INSTALL_DIR/lib64/libbrotlienc.a
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/771299/49d53338-9aa7-497b-903e-d706c245ea7c)

Reply:
1. need disable shared library build, only build static library
2. installed to `lib64` location by default on centos7, but possibly to `lib` location on ubuntu, need specify cmake_lib_dir to make the target library location predictable.